crates/compiler/fmt/fuzz/dict.txt

```diff
@@ -0,0 +1,36 @@
+"if"
+"then"
+"else"
+"when"
+"as"
+"is"
+"expect"
+"dbg"
```

Comment:
I know this PR is only targeting expressions for now, but may be worth adding to this list the following type-related kws, now or later:
- has
- :=
- :
- @

Reply:
Done! I also pointed the command in the readme here to indicated we should just use the dictionary in ../parse/fuzz/dict.txt, so we don't have multiple diverging copies floating around.